src/Pages/SignUpPage/style.ts

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+
+export const StyledWrap = styled.div`
+  height: 100vh;
+  background-color: ${({ theme }) => theme.colors.background};
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const StyledContainer = styled.div`
+  width: 55rem;
+  height: 60rem;
```

Comment:
P5: StyledWrap와 높이를 똑같이 100vh로 맞춰주어야 세로 스크롤이 사라지네요!
```suggestion
  height: 100%;
```